index.html

```diff
@@ -2494,12 +2472,6 @@ <h2>Chunk ordering</h2>
 </td>
 </tr>
 
-<tr>
-<td class="Regular"><a href="#eXIf"><span class="chunk">eXIf</span></a> </td>
-<td class="Regular">No</td>
-<td class="Regular">Before <a href="#idat-image-data"><span class="chunk">IDAT</span></a> </td>
-</tr>
-
```

Comment:
Not sure these were supposed to be removed?

Reply:
You [told me to remove them, and put them in a separate PR](https://github.com/w3c/PNG-spec/pull/105#discussion_r833748466). So I  did.